Cirrious.FluentLayout/AdvancedFluentLayoutExtensions.cs

```diff
@@ -138,6 +138,54 @@ public static IEnumerable<FluentLayout> FullHeightOf(this UIView view, UIView pa
             }
             if (parentView is UIScrollView)
                 yield return previous.Bottom().EqualTo().BottomOf(parentView).Minus(margins.Bottom);
-        }
-    }
+		}
+
+		/// <summary>
+		/// Vertical stack panel constraints with support for children idependent left, right and top margins
+		/// and a multiplier factor for all margins applied. The multiplier can be useful when dealing with ipad screens.
+		/// Example:
+		/// 
+		/// scrollView.AddConstraints(scrollView.AdvancedVerticalStackPanelConstraints(null,
+		///      childrenLeftMargins: new float[] { 15, 0, 15, 0, 0, 15 },
+		///      childrenTopMargins: new float[] { 15, 5, 15, 5, 8, 15, 22, 8, 8, 28, 28 },
+		///      marginMultiplier: 2f,
+		///      views: scrollView.Subviews)
+		/// );
+		/// </summary>
+		public static IEnumerable<FluentLayout> AdvancedVerticalStackPanelConstraints(this UIView parentView,
```

Comment:
The implementation here seems to duplicate a fair bit of code with `VerticalStackPanelConstraints`, would it be possible to share some code there?